A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.html

```diff
@@ -5,11 +5,12 @@
 			<svg width="100%" height="100%" >
 				<defs>
 					<radialGradient id="SAIBacklight" cx="50%" cy="50%" r="75%" >
-						<stop offset="0%" style="stop-color:rgb(13,20,35);stop-opacity:1" />
-						<stop offset="100%" style="stop-color:rgb(13,20,35);stop-opacity:1" />
+						<stop offset="0%" style="stop-color:rgba(4, 4, 5, 1);stop-opacity:1" />
+						<stop offset="100%" style="stop-color:rgba(4, 4, 5, 1);stop-opacity:1" />
```

Comment:
For consistency, why use RGBA for this? The rest of the backlighting uses RGB AFAIK